src/index.ts

```diff
@@ -47,10 +53,40 @@ export async function result(
   repository: string,
   options: IResultOptions = {}
 ): Promise<ScorecardResult> {
+  let formattedRepository = repository;
   const { platform = kDefaultPlatform } = options;
+  const [owner, repo] = repository.replace("@", "").split("/");
+  const octokit = new (Octokit.plugin(restEndpointMethods))({ auth: process.env.GITHUB_TOKEN });
+
+  try {
+    // We try to retrieve the repository with the GitHub API because
+    // Scorecard API is case sensitive, i.e if we pass "nodesecure/cli",
+    // we must reformat it to "NodeSecure/cli"
+    const { data } = await octokit.rest.repos.get({
+      owner,
+      repo
+    });
+    formattedRepository = data.full_name;
```

Comment:
Would be cool to remove Octokit if this is just for one endpoint (introduce a lot of dependency for almost no value).